Some existing build failure
```
[2025-09-22T22:28:55.278Z] FAILED: [code=1] tools/clang/bindings/python/tests/CMakeFiles/check-clang-python /home/build-user/llvm-project/build/tools/clang/bindings/python/tests/CMakeFiles/check-clang-python 
[2025-09-22T22:28:55.278Z] cd /home/build-user/llvm-project/clang/bindings/python && /home/build-user/cmake/bin/cmake -E env CLANG_NO_DEFAULT_CONFIG=1 CLANG_LIBRARY_PATH=/home/build-user/llvm-project/build/lib /usr/bin/python3.10 -m unittest discover
[2025-09-22T22:28:55.278Z] ......................................................................................................................................................................
[2025-09-22T22:28:55.278Z] ======================================================================
[2025-09-22T22:28:55.278Z] FAIL: test_all_variants (tests.cindex.test_enums.TestEnums) [<enum 'CursorKind'>]
[2025-09-22T22:28:55.278Z] Check that all libclang enum values are also defined in cindex.py
[2025-09-22T22:28:55.278Z] ----------------------------------------------------------------------
[2025-09-22T22:28:55.278Z] Traceback (most recent call last):
[2025-09-22T22:28:55.278Z]   File "/home/build-user/llvm-project/clang/bindings/python/tests/cindex/test_enums.py", line 87, in test_all_variants
[2025-09-22T22:28:55.278Z]     self.assertEqual(
[2025-09-22T22:28:55.278Z] AssertionError: {'CXCursor_ForgePtrExpr', 'CXCursor_LastExpr'} variants are missing. Please ensure these are defined in <enum 'CursorKind'> in cindex.py.
[2025-09-22T22:28:55.278Z] 
[2025-09-22T22:28:55.278Z] ----------------------------------------------------------------------
[2025-09-22T22:28:55.278Z] Ran 167 tests in 2.968s
[2025-09-22T22:28:55.278Z] 
[2025-09-22T22:28:55.278Z] FAILED (failures=1)
```

--- a/clang/test/ClangScanDeps/include-tree-working-directory.c
+++ b/clang/test/ClangScanDeps/include-tree-working-directory.c
@@ -20,18 +20,18 @@
 // CACHE-HIT: remark: compile job cache hit
 
 // RUN: cat %t/tu.rsp | sed -E 's|.*"-fcas-include-tree" "(llvmcas://[[:xdigit:]]+)".*|\1|' > %t/tu.casid
-// RUN: clang-cas-test -cas %t/cas -print-include-tree @%t/tu.casid | FileCheck %s -DPREFIX=%/t
+// RUN: clang-cas-test -cas %t/cas -print-include-tree @%t/tu.casid | %PathSanitizingFileCheck --sanitize PREFIX=%/t %s
 
-// CHECK: [[PREFIX]]/t.c llvmcas://
+// CHECK: PREFIX{{/|\\}}t.c llvmcas://
 // CHECK: 1:1 <built-in> llvmcas://
-// CHECK: 2:1 [[PREFIX]]/relative/h1.h llvmcas://
+// CHECK: 2:1 PREFIX{{/|\\}}relative{{/|\\}}h1.h llvmcas://
 // CHECK: Files:
-// CHECK: [[PREFIX]]/t.c llvmcas://
-// CHECK: [[PREFIX]]/relative/h1.h llvmcas://
+// CHECK: PREFIX{{/|\\}}t.c llvmcas://
+// CHECK: PREFIX{{/|\\}}relative{{/|\\}}h1.h llvmcas://
 
 /// Using a different working directory should cache hit as well.
 /// FIXME: Working directory affects some codegen options added by clang driver, preserve them to make sure the cache hit.
-// RUN: sed -e "s|DIR|%/t|g" %t/cdb2.json.template > %t/cdb2.json
+// RUN: sed -e "s|DIR|%/t|g" %t/cdb2.json.template | sed -e "s|SEP|%{fs-sep-yaml}|g" > %t/cdb2.json
 // RUN: clang-scan-deps -compilation-database %t/cdb2.json -format experimental-include-tree-full -cas-path %t/cas \
 // RUN:   > %t/deps2.json
 // RUN: %deps-to-rsp %t/deps2.json --tu-index 0 > %t/tu2.rsp
@@ -48,7 +48,7 @@
 //--- cdb2.json.template
 [{
   "directory": "DIR/other",
-  "command": "clang -c DIR/t.c -I DIR/relative -working-directory DIR/other -o DIR/t2.o -MD -serialize-diagnostics DIR/t2.dia -fdebug-compilation-dir=DIR -fcoverage-compilation-dir=DIR",
+  "command": "clang -c DIRSEPt.c -I DIRSEPrelative -working-directory DIR/other -o DIR/t2.o -MD -serialize-diagnostics DIR/t2.dia -fdebug-compilation-dir=DIR -fcoverage-compilation-dir=DIR",
   "file": "DIR/t.c"
 }]
 

--- a/llvm/utils/lit/lit/TestRunner.py
+++ b/llvm/utils/lit/lit/TestRunner.py
@@ -1501,6 +1501,7 @@ def getDefaultSubstitutions(test, tmpDir, tmpBase, normalize_slashes=False):
             ("%{fs-src-root}", pathlib.Path(sourcedir).anchor),
             ("%{fs-tmp-root}", pathlib.Path(tmpBase).anchor),
             ("%{fs-sep}", os.path.sep),
+            ("%{fs-sep-yaml}", "\\\\\\\\\\\\\\\\" if kIsWindows else "/"),
         ]
     )
 

